Models/Entities/PSPDatabaseContext.cs

```diff
@@ -7,10 +7,27 @@ public class PSPDatabaseContext : DbContext
     public PSPDatabaseContext(DbContextOptions<PSPDatabaseContext> options)
     : base(options)
     {
+        ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
+        Database.EnsureCreated();
+    }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
requirements had no mention of creating user or etc. Should we just keep seeds, or create separate controller for user related endpoints?

Reply:
im for keeping seeds xd

Reply:
yep sounds good